### ISSUE TITLE
`*Formatter` tests fail on pl_PL locale

### ISSUE THREAD

**Issue by Saviq** [canonical/multipass#204]
```
$ LC_ALL=pl_PL.UTF-8 ./build/bin/multipass_tests
# [...]
[ FAILED ] 6 tests, listed below:
[ FAILED ] TableFormatter.single_instance_info_output
[ FAILED ] TableFormatter.multiple_instances_info_output
[ FAILED ] JsonFormatter.single_instance_info_output
[ FAILED ] JsonFormatter.multiple_instances_info_output
[ FAILED ] YamlFormatter.single_instance_info_output
[ FAILED ] YamlFormatter.multiple_instances_info_output
```

The reason is formatting floats, which uses `,` here, but `.` is expected.

**Comment by townsend2010** [comment]
As a data point, I don't see these tests failing locally for me.

**Comment by albaguirre** [comment]
Fixed via #209
